Extension/lib/content-script/assistant/js/adguard-rules-constructor.js

```diff
@@ -124,6 +126,45 @@ var AdguardRulesConstructorLib = (function (api) {
         return blockUrlRuleText;
     };
 
+    var getUrlBlockAttribute = function (element) {
+        for (var i = 0; i < URLBLOCK_ATTRIBUTES.length; i++) {
+            var attr = URLBLOCK_ATTRIBUTES[i];
+            var value = element.getAttribute(attr);
+            if (value) {
```

Comment:
You should also check that value is a valid URL.
It may be "data:image" for instance.